### ISSUE TITLE
Scanning Broken after 1st 100%

### ISSUE THREAD

**Issue by primaxius** [TheZ3ro/pkmngo-map#19]
Scans properly to 100% then once it starts again it does not return any data.

`[+] Scan: 94.0 %
[+] (10) Caterpie is visible at (53.4250752135, -117.537325141) for 78 seconds
[+] (46) Paras is visible at (53.4252167466, -117.539687748) for 601 seconds
[+] (16) Pidgey is visible at (53.4261849495, -117.537272356) for 378 seconds
[-] Two Pidgey at the same location (53.4261849495,-117.537272356)
[+] (32) Nidoran M is visible at (53.4242959031, -117.535687388) for -194940 seconds
[+] Scan: 95.0 %
[+] (48) Venonat is visible at (53.4237079844, -117.538043749) for -194941 seconds
[+] (48) Venonat is visible at (53.4225441684, -117.540876737) for -194942 seconds
[+] Scan: 96.0 %
[+] Scan: 97.0 %
[+] Scan: 98.0 %
[+] Scan: 99.0 %
[+] Scan: 100.0 %
[+] Scan: 1.0 %
[+] Scan: 2.0 %
[+] Scan: 3.0 %
[+] Scan: 4.0 %
[+] Scan: 5.0 %
[+] Scan: 6.0 %
[+] Scan: 7.0 %
[+] Scan: 8.0 %
[+] Scan: 9.0 %
[+] Scan: 10.0 %
[+] Scan: 11.0 %
[+] Scan: 12.0 %
[+] Scan: 13.0 %
[+] Scan: 14.0 %
[+] Scan: 15.0 %
[+] Scan: 16.0 %
[+] Scan: 17.0 %
[+] Scan: 18.0 %
[+] Scan: 19.0 %
[+] Scan: 20.0 %
[+] Scan: 21.0 %
[+] Scan: 22.0 %
[+] Scan: 23.0 %
[+] Scan: 24.0 %
[+] Scan: 25.0 %
[+] Scan: 26.0 %
[+] Scan: 27.0 %
[+] Scan: 28.0 %
[+] Scan: 29.0 %
[+] Scan: 30.0 %
[+] Scan: 31.0 %
[+] Scan: 32.0 %
[+] Scan: 33.0 %
[+] Scan: 34.0 %`

**Comment by TheZ3ro** [comment]
Duplicate of https://github.com/TheZ3ro/pkmngo-map/issues/16

**Comment by padmanek** [comment]
After roughly 30-40min the script stops finding new pokemons and the maps eventually becomes empty ( when existing pokemon expire). It looks like this in the command line:
http://i.imgur.com/8E897cl.png

**Comment by TheZ3ro** [comment]
Yes, I know, you can talk about it here: https://github.com/TheZ3ro/pkmngo-map/issues/16

I will close this

**Comment by TheZ3ro** [comment]
Found out that the Step scanning is ok but still don't find new pokemon.

i'm working on this

**Comment by primaxius** [comment]
https://github.com/AHAAAAAAA/PokemonGo-Map <--they fixed it maybe have a look
